tests/bundle/Core/DependencyInjection/Compiler/AggregateFieldValueMapperPassTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Tests\Bundle\Core\DependencyInjection\Compiler;
+
+use eZ\Publish\Core\Base\Container\Compiler\Search\AggregateFieldValueMapperPass;
+use eZ\Publish\SPI\Search\FieldType\BooleanField;
+use Matthias\SymfonyDependencyInjectionTest\PhpUnit\AbstractCompilerPassTestCase;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+use Symfony\Component\DependencyInjection\Reference;
+
+class AggregateFieldValueMapperPassTest extends AbstractCompilerPassTestCase
```

Comment:
```suggestion
final class AggregateFieldValueMapperPassTest extends AbstractCompilerPassTestCase
```

and in all other introduced tests classes

Reply:
Fixed: b06aea0dcd and 05d2d55ac4 for the other